### PR TITLE
api refactor, rename options

### PR DESCRIPTION
**Pull request by dtassone**

### CODE REVIEW

**Comment by dtassone** [comment]
Sourcemaps not working :(

File: .eslintrc.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ module.exports = {
 
     // TODO
     'react/prop-types': 'off', // For public API
-    'import/prefer-default-export': 'warn',
+    'import/prefer-default-export': 'off',
```
Comment (dtassone) [inline .eslintrc.js:87]: Creates too many warning. 
Don't like default as, it breaks refactoring tool and a var can be named x in one import and y in another. Why is that great? And if you decide to rename it Z it won't propagate...

Comment (oliviertassinari) [reply .eslintrc.js:87]: In this case, please contribute in https://github.com/mui-org/material-ui/issues/21862 and leave the rule in place until we find a resolution.

**Comment by dtassone** [comment]
Need to publish the package for it to work in storybook

File: packages/grid/data-grid/src/data-grid.tsx [hunk 2]
```diff
@@ -55,7 +56,7 @@ export const DataGrid: React.FC<DataGridProps> = React.memo(function DataGrid(
       loading={props.loading}
       options={internalOptions}
       licenseStatus="Valid"
-      className={`data-grid ${props.className || ''}`}
+      className={classnames('data-grid', props.className)}
```
Comment (oliviertassinari) [inline packages/grid/data-grid/src/data-grid.tsx:59]: +1

**Pull request merged**